FlagsmithClient/src/test/java/com/flagsmith/TraitsTests.kt

```diff
@@ -83,7 +83,7 @@ class TraitsTests {
         mockServer.mockResponseFor(MockEndpoint.SET_TRAIT)
         runBlocking {
             val result =
-                flagsmith.setTraitSync(Trait(key = "set-from-client", value = "12345"), "person")
+                flagsmith.setTraitSync(Trait(key = "set-from-client", traitValue = "12345"), "person")
```

Comment:
Why has this changed?

Reply:
It was due to my previous changes, it has been reverted now.